src/endpoints/resources.js

```diff
@@ -199,6 +203,12 @@ const parseDate = (dateString) => {
   return date
 }
 
+const parseJsonld = (jsonldString) => {
+  datasetFromJsonld(jsonldString).catch((err) => {
+    throw createError(400, 'Bad Request', {details: `Unsparseable JSON-LD: ${err}`})
```

Comment:
the tests know this is never tripped ... but I can't figure out how to get more context in the details for the 400 error.